src/main/java/com/google/cloud/tools/gradle/appengine/base/extension/Deploy.java

```diff
@@ -15,7 +15,7 @@
  *
  */
 
-package com.google.cloud.tools.gradle.appengine.model;
+package com.google.cloud.tools.gradle.appengine.base.extension;
```

Comment:
I'm not a big fan of 'base' as a package name.  Maybe 'common' or if it's for a provided framework maybe 'framework'?